core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1031,20 +1030,46 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def addGoogleProjectLabel(googleProject: GoogleProjectId, key: String, value: String): Future[Unit] = {
+    implicit val service: GoogleInstrumentedService.Value = GoogleInstrumentedService.CloudResourceManager
+
+    val cloudResourceManagerServiceAccountCredential = getCloudResourceManagerServiceAccountCredential
+    val cloudResourceManager = getCloudResourceManager(cloudResourceManagerServiceAccountCredential)
+    for {
+      project <- getGoogleProject(googleProject)
+      _ = project.getLabels.put(key, value)
+      _ <- executeGoogleRequestWithRetry(cloudResourceManager.projects().update(googleProject.value, project))
+    } yield {
+
+    }
+  }
+
+  override def updateGoogleProjectName(googleProject: GoogleProjectId, name: String): Future[Unit] = {
```

Comment:
suggestion: rename or add comment that this is the google project display name

Reply:
I have a slight preference towards commenting.  Since this is the `GoogleServicesDAO`, it is specifically about getting/setting values in Google land and this _is_ what the field is called on the Google Project.  That said, any time I need to do something with the Google Project Name or ID I reference this page:  https://cloud.google.com/resource-manager/docs/creating-managing-projects#before_you_begin which disambiguates `Project ID`, `Project Name`, and `Project Number`.  Some commenting here would be helpful and perhaps a link to the Google documentation.  Yeah, it might move, but it's the canonical source for what these things are and what they mean right now, and that's the best we can do.